homeassistant/components/lifx_cloud/scene.py

```diff
@@ -68,6 +68,19 @@ async def async_setup_platform(
     _LOGGER.error("HTTP error %d on %s", scenes_resp.status, url)
 
 
+def map_to_lifx_request(service_args):
+    """Map options from home assistant service call to lifx request."""
+    body = {}
+
+    if (
+        ATTR_TRANSITION in service_args
+        and service_args.get(ATTR_TRANSITION) is not None
+    ):
+        body["duration"] = service_args.get(ATTR_TRANSITION)
+
+    return body
```

Comment:
```suggestion
    if duration := service_args.get(ATTR_TRANSITION) is not None:
        body["duration"] = duration

    return body
```
If the value in ATTR_TRANSITION cannot be 0, then `is not None` is not needed.

Reply:
It could be zero, for example, if you wanted to override a scene's transition time stored in LIFX and have it apply instantly.